aws-ecr-repository/src/test/java/software/amazon/ecr/repository/ConfigurationTest.java

```diff
@@ -0,0 +1,22 @@
+package software.amazon.ecr.repository;
+
+import java.util.Map;
+import org.junit.jupiter.api.Test;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class ConfigurationTest {
+
+    @Test
+    void testMergeDuplicateKeys() {
```

Comment:
Would like to start being more descriptive on test naming. We can also remove the 'test' part, I think, so something like '{condition}_{expectedBehavior}'

Reply:
Yes. Updated as suggested.